### PR TITLE
ScTmplParam Type update

### PR DESCRIPTION
**Pull request by amityadav0**
https://github.com/scale-it/algorand-builder/issues/243

### CODE REVIEW

File: packages/algob/src/types.ts [hunk 1]
```diff
@@ -545,6 +545,10 @@ export interface StrMap {
   [key: string]: string
 }
 
+export interface ScParamsMap {
+  [key: string]: string | bigint
```
Comment (ratik21) [inline packages/algob/src/types.ts:549]: Shouldn't we include numbers as well? We recommend `bigint`, but i think we should have `number` as well. Right?

Comment (amityadav0) [reply packages/algob/src/types.ts:549]: https://github.com/scale-it/algorand-builder/issues/243#issuecomment-781930008

Comment (ratik21) [reply packages/algob/src/types.ts:549]: I am not sure if it means to remove the support for `number` and use only `bigint`. Ignore if i misunderstood.

File: packages/algob/src/types.ts [hunk 2]
```diff
@@ -545,6 +545,10 @@ export interface StrMap {
   [key: string]: string
 }
 
+export interface ScParamsMap {
```
Comment (robert-zaremba) [inline packages/algob/src/types.ts:548]: How about renaming to `SCParams`?

**Review by robert-zaremba — APPROVED** [review]
Left one suggestion

File: packages/algob/src/types.ts (hunk 2 above)
Comment (amityadav0) [reply packages/algob/src/types.ts:548]: ok

**Pull request merged**